front/app/containers/UsersEditPage/messages.ts

```diff
@@ -30,6 +30,15 @@ export default defineMessages({
     id: 'app.containers.UsersEditPage.messageSuccess',
     defaultMessage: 'Your profile has been saved.',
   },
+  profanityError: {
+    id: 'app.containers.UsersEditPage.profanityError',
+    defaultMessage:
+      'You may have used one or more words that are considered profanity by {guidelinesLink}. Please alter your text to remove any profanities that might be present.',
+  },
+  guidelinesLinkText: {
+    id: 'app.containers.UsersEditPage.guidelinesLinkText',
+    defaultMessage: 'our guidelines',
+  },
```

Comment:
Is it more efficient to create a new message instead of reusing the one from idea editing?